src/PSRule.Rules.Azure/rules/Azure.PublicIP.Rule.ps1

```diff
@@ -36,3 +36,46 @@ Rule 'Azure.PublicIP.DNSLabel' -Type 'Microsoft.Network/publicIPAddresses' -If {
     # End with a letter or number.
     $Assert.Match($TargetObject, 'Properties.dnsSettings.domainNameLabel', '^[a-z][a-z0-9-]{1,61}[a-z0-9]$', $True);
 }
+
+# Synopsis: Public IP addresses deployed with Standard SKU should use availability zones in supported regions for high availability.
+Rule 'Azure.PublicIP.AvailabilityZone' -Type 'Microsoft.Network/publicIPAddresses' -If { IsStandardPublicIP } -Tag @{ release = 'GA'; ruleSet = '2021_09'; } {
```

Comment:
Let's move this to the upcoming 2021_12 ruleset because 2021_09 is already released.

Reply:
No worries @BernieWhite. Have moved to 2021_12 ruleset. Also created a baseline test for this ruleset as well.